syncthing.if

```diff
@@ -0,0 +1,55 @@
+########################################
+##	<summary>
+##	Role access for Syncthing
+##	</summary>
+##	<param name="role">
+##		<summary>
+##			Role allowed access
+##		</summary>
+##	</param>
+##	<param name="domain">
+##		<summary>
+##			User domain for the role
+##		</summary>
+##	</param>
+#
+interface(`syncthing_role', `
+
+	gen_require(`
+		attribute_role syncthing_roles;
+		type syncthing_t, syncthing_exec_t, syncthing_config_home_t;
+	')
+
+	# Declarations
+	roleattribute $1 syncthing_roles;
+
+	# Policy
+	domtrans_pattern($2, syncthing_exec_t, syncthing_t)
+')
```

Comment:
This template should provide access to manage and relabel syncthing_config_home_t type content. Else the login shell domain is not allowed to manage and relabel ~/.config/synchting